governance/votes/Executive vote - December 3, 2021.md

```diff
@@ -0,0 +1,136 @@
+* [Flat Kick Incentive (`tip`)](https://community-development.makerdao.com/en/learn/governance/param-flat-kick-incentive): 300 DAI
+
+Please review the following forum threads containing information about GUNIV3DAIUSDC2-A to inform your position before voting.
+* [Proposal Thread](https://forum.makerdao.com/t/g-uni-dai-usdc-0-01-tier-onboard-g-uni-for-univ3s-new-0-01-fee-tier/11738)
+* [Risk Core Unit Evaluation](https://forum.makerdao.com/t/g-uni-dai-usdc-collateral-onboarding-risk-evaluation/9719)
+* [Protocol Engineering Core Unit Evaluation](https://forum.makerdao.com/t/guni-dai-usdc-erc20-token-smart-contract-technical-assessment/9139)
+* [Oracles Team Evaluation](https://forum.makerdao.com/t/guni-dai-usdc-collateral-onboarding-oracle-assessment-mip10c3-sp41/10268)
+
+### Delegate Compensation Payments - November
+
+As per this successful [governance poll](https://vote.makerdao.com/polling/QmbvuhYH?network=mainnet#poll-detail), and the ongoing trial of delegate compensation, a total of 35,783 DAI will be distributed to four recognized delegates as detailed below if this executive proposal passes.
+
+- **12,000 DAI** transferred to [FlipFlopFlap Delegate LLC](https://vote.makerdao.com/address/0xaf8aa6846539033eaf0c3ca4c9c7373e370e039b?network=mainnet) at [0x688d508f3a6B0a377e266405A1583B3316f9A2B3](https://etherscan.io/address/0x688d508f3a6B0a377e266405A1583B3316f9A2B3).
+- **12,000 DAI** transferred to [Feedblack Loops LLC](https://vote.makerdao.com/address/0x845b36e1e4f41a361dd711bda8ea239bf191fe95?network=mainnet) at [0x80882f2A36d49fC46C3c654F7f9cB9a2Bf0423e1](https://etherscan.io/address/0x80882f2A36d49fC46C3c654F7f9cB9a2Bf0423e1).
+- **8,093 DAI** transferred to [ultraschuppi](https://vote.makerdao.com/address/0xb21e535fb349e4ef0520318acfe589e174b0126b?network=mainnet) at [0x89C5d54C979f682F40b73a9FC39F338C88B434c6](https://etherscan.io/address/0x89C5d54C979f682F40b73a9FC39F338C88B434c6).
+- **3,690 DAI** transferred to [Field Technologies Inc.](https://vote.makerdao.com/address/0xad2fda5f6ce305d2ced380fdfa791b6a26e7f281?network=mainnet) at [0x0988E41C02915Fe1beFA78c556f946E5F20ffBD3](https://etherscan.io/address/0x0988E41C02915Fe1beFA78c556f946E5F20ffBD3).
+
+Details on the breakdown of delegate compensation for the month of November can be found on the MakerDAO forum [here](https://forum.makerdao.com/t/delegate-compensation-breakdown-november-2021/11979).
+
+### GovComms Core Unit One Time Payment
+
+As per this successful [governance poll](https://vote.makerdao.com/polling/Qmc7Necw?network=mainnet#poll-detail), a total of 27,058 DAI will be transferred to the [Governance Communications Core Unit (COM-001)](https://mips.makerdao.com/mips/details/MIP39c2SP8) at [0x1eE3ECa7aEF17D1e74eD7C447CcBA61aC76aDbA9](https://etherscan.io/address/0x1eE3ECa7aEF17D1e74eD7C447CcBA61aC76aDbA9) if this executive proposal passes.
+
```

Comment:
Link to forum post seems important here too for context

Reply:
Yep